test/validation/api/ipsec/ipsec_test_out.c

```diff
@@ -1591,6 +1591,150 @@ static void test_test_sa_update_seq_num(void)
 	printf("\n  ");
 }
 
+#define SOFT_LIMIT_PKT_CNT 1024
+#define HARD_LIMIT_PKT_CNT 2048
+#define DELTA_PKT_CNT 320
+
+static void test_out_ipv4_esp_sa_expiry(enum ipsec_test_sa_expiry expiry)
+{
+	int test_packet_len = pkt_ipv4_icmp_0.len - pkt_ipv4_icmp_0.l3_offset;
```

Comment:
Maybe rename this to e.g. byte_count_per_packet as it is not really test packet length?

Reply:
Ack. Renamed the variable it to byte_count_per_packet.